config/config.xsd

```diff
@@ -345,11 +347,45 @@
                     </xs:restriction>
                 </xs:simpleType>
             </xs:attribute>
+            <xs:attribute name="audio-layout" type="string"/>
```

Comment:
What are these for?

Reply:
They are used in `import.js` to switch to e.g. structured layout. So there is only one import script which selects the layout function instead of replacing the import file. Obviously I forgot to add documentation for it. Currently only `audio-layout` is referenced, but I added the others to make things symmetric.

Reply:
See scripts/js/import.js